keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -236,11 +240,14 @@ Once the tests show that the mirrors are good, we can adapt the official documen
 
 #### Alpha -> Beta Graduation
 
+- [ ] [krel] creates and publishes deb and rpm packages of Kubernetes components
+- [ ] post-publish tests are written and run as part of the release process
+- [ ] nightly builds will be built and published on a daily basis using [krel] which will be improved to take over this task from [kubepkg] making use of [pre-existing periodic jobs] (https://github.com/kubernetes/test-infra/blob/97cb34fa9e2bfc4af35de3e561cb9fc5a1094da1/config/jobs/kubernetes/sig-release/kubernetes-builds.yaml#L120-L166)
+- [ ] documentation written checked to be complete and correct. 
 
-- [anago] [creates packages][pkg-gen-kep] and published those packages as part of the release process
-- post-publish tests are in places and run as part of the release process
-- nightlies will be build and published on a daily basis
-- documentation is in place
+
+[kubepgk]: https://github.com/kubernetes/release/tree/master/cmd/kubepkg 
+[krel]: https://github.com/kubernetes/release/blob/master/docs/krel/README.md 
 
 #### Beta -> GA Graduation
 
```

Comment:
GA Graduation should probably require:
- Removing remaining documentation for the old google hosted packages

Reply:
Plus deprecation policy + comms plan

Reply:
I've added headers for Documentation removal, Deprecation policy, and Comns plans.

Reply:
> Plus deprecation policy + comms plan

I think we need to do this much earlier. We should probably have some initial deprecation policy ideas outlined in this KEP. For comms, I think we should start communicating as early as possible, probably when graduating to beta. For example:

* Announce that we started efforts on publishing packages to community infra, provide semi-frequent updates to the commucnity
* Once we graduate to beta, start promoting new infrastructure via all communication channels. Ask users to give it a try and provide us feedback
* When graduating to GA, start the deprecation period and communicate that via all communication channels, including CNCF

Additionally, I think we might need some sort of survey in beta or when graduating to GA. We should collect information from users did they tested new infra, how it works for them, and how much effort is it to migrate to new infra for them.